samples/code-samples/ChangeFeedProcessor/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.DocumentDB" version="1.17.0" targetFramework="net461" />
-  <package id="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" version="1.1.1" targetFramework="net461" />
+  <package id="Microsoft.Azure.DocumentDB" version="1.21.1" targetFramework="net461" />
+  <package id="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" version="1.0.0" targetFramework="net461" />
```

Comment:
Wrong version, maybe a file that got skipped from commit? Can you please try building and running from a clean fork?